src/utils/formatBalance.ts

```diff
@@ -15,6 +15,10 @@ export const getBalanceAmount = (amount: BigNumber, decimals = 18) => {
   return new BigNumber(amount).dividedBy(BIG_TEN.pow(decimals))
 }
 
+export const getEthersBalanceAmount = (amount: ethers.BigNumber, decimals = 18) => {
+  return ethers.BigNumber.from(amount).div(ethers.BigNumber.from(10).pow(decimals))
+}
```

Comment:
A couple things:

1. The function accepts a BigNumber then converts it into another BigNumber.

2. Please check https://docs.ethers.io/v5/api/utils/display-logic/

Reply:
Done, I removed the function and use ethers formatUtils now